backend/routes/prisonerProfileRouter.ts

```diff
@@ -190,6 +190,14 @@ const controller = ({
 
   // @ts-expect-error ts-migrate(2345) FIXME: Argument of type '{ oauthApi: any; prisonApi: any;... Remove this comment to see the full error message
   router.get('/cell-history', prisonerCellHistory({ oauthApi, prisonApi, logError }))
+
+  router.get(
+    '/cell-history',
+    prisonerProfileRedirect({
+      path: '/location-details',
+      handler: prisonerCellHistory({ oauthApi, prisonApi }),
```

Comment:
Should we keep the `logError` param in the call?

Reply:
Thanks for taking a look @davidmiddletonmoj, I have added the logError back in